www/podj00/sp2/assets/js/main.js

```diff
@@ -31,22 +36,21 @@ $(document).ready(() => {
     let mockedResponse = {};
     let mockedOneCountry = {};
 
-    $('input[type="checkbox"]').on('change', (e) => {
-        //only 1 input (clicked one) can be checked
-        $('input[type="checkbox"]').not(e.currentTarget).prop('checked', false);
-        //cannot uncheck - at least one has to be checked
-        $(e.currentTarget).prop('checked', true);
+    $('input[type="radio"]').on('change', (e) => {
+        //only 1 radio can be checked
+        $('input[type="radio"]').not(e.currentTarget).prop('checked', false);

```

Comment:
tady jste asi nepochopil, to ze ma byt vybrana pouze jedna moznost je defaultni chovani radio buttonu, takze neni potreba implementovat udalost change..